File: CPKINGDOM/CPKINGDOM.Portal/ClientApp/src/app/models/JResponse.ts
export class JResponse {
    success: boolean;
    message: string;
    errorCode: string;
    data: any;
    total: number;
    constructor() {
        this.success = true;
        this.data = {};
        this.total = 0;
        this.errorCode = '';
        this.message = '';
    }
}

File: CPKINGDOM/CPKINGDOM.Core/Models/JResponse.cs
namespace CPKINGDOM.Core.Models
{
    public class JResponse
    {

        public bool Success { get; set; } = true;
        public string Message { get; set; } = "";
        public string ErrorCode { get; set; } = "";
        public object Data { get; set; }
        public int Total { get; set; } = 0;
    }
}
